### PR TITLE
test(CreateSidePanel): add default state AVT test

### LINKED ISSUE #4663: Add default state AVT check for CreateSidePanel
### Acceptance Criteria

See parent issue https://github.com/carbon-design-system/ibm-products/issues/4626

### PR DESCRIPTION
**Pull request by anamikaanu96**
Closes #4663 

Created e2e/components/CreateSidePanel/CreateSidePanel-test.avt.e2e.js

### CODE REVIEW

**Review by emyarod — COMMENTED** [review]
it looks like the avt suite is still timing out for some reason, even after rerunning

https://github.com/carbon-design-system/ibm-products/actions/runs/9109254784/job/25041736070?pr=5247
https://github.com/carbon-design-system/ibm-products/actions/runs/9109254784/job/25055608101?pr=5247

File: cspell.json [hunk 1]
```diff
@@ -85,6 +85,7 @@
     "combobox",
     "combobutton",
     "createmodal",
+    "createsidepanel",
```
Comment (IgnacioBecerra) [inline cspell.json:88]:
```suggestion
```
We don't need to add any files to the cspell file, as #5202 takes care of the avt files.

**Review by IgnacioBecerra — CHANGES_REQUESTED** [review]
Other than my comment, everything seems good, not sure why the runner is failing like Andrew mentioned but otherwise things do look good.

**Review by annawen1 — APPROVED** [review]
LGTM!

**Review by ariellalgilmore — APPROVED** [review]
LGTM!

**Review by elycheea — APPROVED** [review]

**Pull request merged**